### PR TITLE
[zh-cn] sync cluster-administration/flow-control.md system-metrics.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/cluster-administration/flow-control.md
content/zh-cn/docs/concepts/cluster-administration/system-metrics.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/cluster-administration/flow-control.md [hunk 1]
```diff
@@ -80,48 +80,38 @@ APF 适用于 **watch** 请求。当 APF 被禁用时，**watch** 请求不受 `
...
-你可以在启动 `kube-apiserver` 时，添加以下命令行标志来禁用此功能门控及 API Beta 组：
+(a) 稳定的 `v1` 版本，在 1.29 中引入，默认启用；
+(b) `v1beta3` 版本，默认被启用，在 1.29 中被弃用。
+你可以通过添加以下内容来禁用 Beta 版的 `v1beta3` API 组：
 
 ```shell
 kube-apiserver \
---feature-gates=APIPriorityAndFairness=false \
---runtime-config=flowcontrol.apiserver.k8s.io/v1beta2=false,flowcontrol.apiserver.k8s.io/v1beta3=false \
+--runtime-config=flowcontrol.apiserver.k8s.io/v1beta2=false \
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/cluster-administration/flow-control.md:106]:
```suggestion
--runtime-config=flowcontrol.apiserver.k8s.io/v1beta3=false \
```

??

File: content/zh-cn/docs/concepts/cluster-administration/system-metrics.md [hunk 2]
```diff
@@ -352,12 +351,35 @@ Here is an example:
...
+此外，`cardinality_enforcement_unexpected_categorizations_total`
+元指标记录基数执行期间意外分类的计数，即每当遇到允许列表约束不允许的标签值时。
+
 ## {{% heading "whatsnext" %}}
 
 <!--
 * Read about the [Prometheus text format](https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md#text-based-format)
   for metrics
-* Read about the [Kubernetes deprecation policy](/docs/reference/using-api/deprecation-policy/#deprecating-a-feature-or-behavior)
+* Read about the [Kubernetes deprecation policy](/docs/reference/using-api/deprecation-policy/#deprecating-a-feature-or-behavior)* Read about the [Kubernetes deprecation policy](/docs/reference/using-api/deprecation-policy/#deprecating-a-feature-or-behavior)
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/cluster-administration/system-metrics.md:382]: ?

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**